permits/views.py

```diff
@@ -1171,6 +1172,33 @@ def permit_request_classify(request, permit_request_id, approve):
             }
             services.send_email_notification(data)
 
+            # Notify the services
+            works_object_types_to_notify = permit_request.works_object_types.filter(
+                notify_services=True
+            )
+
+            if works_object_types_to_notify.exists():
+                mailing_list = []
+                for emails in works_object_types_to_notify.values_list(
+                    "services_to_notify", flat=True
+                ):
+                    emails_addresses = emails.replace(" ", "").split(",")
```

Comment:
There's a `strip()` function on strings which does that replacement on extremities. Is that what you want?

We shouldn't replace spaces inside the email, which might be valid in very rare cases (like `foo\ bar@space.com`).

Reply:
Ok, better to use strip() after....
I added the help text, but just to avoid human error for newlines, we are covered by changing to this line to: 
`emails_addresses = emails.replace("\n", ",").split(",")`